BookingsApi/BookingsApi.IntegrationTests/Database/Commands/RemoveParticipantFromHearingCommandTests.cs

```diff
@@ -78,17 +74,36 @@ public async Task Should_remove_participant_with_endpoint_from_hearing()
         {
             var seededHearing = await Hooks.SeedVideoHearing();
             TestContext.WriteLine($"New seeded video hearing id: {seededHearing.Id}");
-            _newHearingId = seededHearing.Id;
             var beforeCount = seededHearing.GetParticipants().Count;
 
             var endpoint = seededHearing.GetEndpoints().First(ep => ep.DefenceAdvocate != null);
-            await _commandHandler.Handle(new RemoveParticipantFromHearingCommand(seededHearing.Id, endpoint.DefenceAdvocate));
+            await _commandHandler.Handle(
+                new RemoveParticipantFromHearingCommand(seededHearing.Id, endpoint.DefenceAdvocate));
 
             var returnedVideoHearing =
                 await _getHearingByIdQueryHandler.Handle(new GetHearingByIdQuery(seededHearing.Id));
             var afterCount = returnedVideoHearing.GetParticipants().Count;
 
             afterCount.Should().BeLessThan(beforeCount);
         }
+
+        [Test]
+        public async Task Should_Remove_ParticipantLink_When_Participant_Is_Removed()
+        {
+            var seededHearing = await Hooks.SeedVideoHearing(null, false, BookingStatus.Booked, 0, false, true);
+            TestContext.WriteLine($"New seeded video hearing id: {seededHearing.Id}");
+
+            var participantWithALink = seededHearing.Participants.First(x => x.LinkedParticipants.Any());
+            await _commandHandler.Handle(
+                new RemoveParticipantFromHearingCommand(seededHearing.Id, participantWithALink));
+
+            var links = seededHearing.Participants.Select(x =>
```

Comment:
You should be re-querying the db here and asserting the `linkedparticipants` property is empty